AppCenterIdentity/AppCenterIdentityTests/MSIdentityTests.m

```diff
@@ -346,6 +346,82 @@ - (void)testNotCacheInvalidData {
   [ingestionMock stopMocking];
 }
 
-// TODO add tests to cover login.
+- (void)testLoginAcquiresToken {
+
+  // If
+  MSIdentity *service = [MSIdentity sharedInstance];
+  id clientApplicationMock = OCMPartialMock([MSALPublicClientApplication alloc]);
+  service.clientApplication = clientApplicationMock;
+  service.identityConfig = [MSIdentityConfig new];
+  service.identityConfig.identityScope = @"fake";
+  id identityMock = OCMPartialMock(service);
+  OCMStub([identityMock sharedInstance]).andReturn(identityMock);
+  OCMStub([identityMock canBeUsed]).andReturn(YES);
+  OCMStub([clientApplicationMock acquireTokenForScopes:OCMOCK_ANY completionBlock: OCMOCK_ANY]).andDo(nil);
```

Comment:
I think you can improve this test and verify accessToken as well. In `andDo`, call `completionBlock` with mock `accessToken` and then verify the accessToken` with `service.accessToken`.